client/network/src/protocol/notifications/behaviour.rs

```diff
@@ -2152,38 +2153,69 @@ mod tests {
 		}
 	}
 
-	fn development_notifs() -> (Notifications, crate::peerset::PeersetHandle) {
-		let (peerset, peerset_handle) = {
-			let mut sets = Vec::with_capacity(1);
+	#[derive(Debug)]
+	struct MockPeerStore {}
+
+	impl PeerStoreProvider for MockPeerStore {
```

Comment:
This should be provided by `peer_store.rs` in `mock.rs` or something because now we have duplicates. One in here and one in `requests_responses.rs`.

Reply:
Done